base_layer/core/src/chain_storage/postgres/migrations/2020-04-07-14-49_tx_outputs/up.sql

```diff
@@ -0,0 +1,15 @@
+create table tx_outputs(
+    hash TEXT NOT NULL PRIMARY KEY,
+    features_flags SMALLINT NOT NULL,
+    features_maturity BIGINT NOT NULL,
+    commitment TEXT NOT NULL,
+    proof BYTEA NULL,
+    spent BIGINT NOT NULL DEFAULT 0,
+    created_at TIMESTAMP NOT NULL DEFAULT current_timestamp,
+);
+
+create index index_tx_outputs_hash on tx_outputs(hash);
+create index index_tx_outputs_spent on tx_outputs(boolean);
```

Comment:
Indices on boolean columns are pretty pointless.

Reply:
changed to hash field